docs/REVIEWING.md

```diff
@@ -37,32 +37,34 @@ following items should also be checked:
 - [ ] Does it comply with the [instrumentation spec](specs/instrumentation.md)?
 
 
-### Sink review checklist
+### Checklist - new sink
 
 This checklist is specific for Vector's sink code.
 
-## Logic
+#### Logic
 
 - [ ] Does it work? Do you understand what it is supposed to be doing?
 - [ ] Does the retry logic make sense?
 - [ ] Are the tests testing that the sink is emitting the correct metrics?
 - [ ] Are there integration tests?
 
-## Code structure
+#### Code structure
 
-- [ ] Is it using the sink prelude?
+- [ ] Is it using the sink prelude (`use crate::sinks::prelude::*`)?
+- [ ] Is the sink a stream based sink?
+      Check that the return value from `SinkConfig::build` is the return from `VectorSink::from_event_streamsink`.
 - [ ] Is it gated by sensible feature flags?
 - [ ] Is the code modularized into `mod.rs`, `config.rs`, `sink.rs`,  `request_builder.rs`, `service.rs`
 - [ ] Does the code follow our [style guidelines].
 
-## Documentation
+#### Documentation
 
-- [ ] Look at the doc preview on Netlify. Does it look good? 
+- [ ] Look at the doc preview on Netlify. Does it look good?
```

Comment:
Just noticed this sorry- Are there any specifics to ensure it looks good? Like, read through the new docs as if you were a new user trying to use that sink? Or is the message to ensure it doesn't show flag any errors?

Reply:
I was just thinking mainly looking for obvious errors, eg. the cue file doesn't include the base file properly?